libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -180,12 +184,8 @@ export const reviewCreateTab = (
         description: 'Button text for creating the workflow',
       })
     ),
-    primaryButtonOnClick: isCreated
-      ? () => TemplateService()?.openBladeAfterCreate(workflowName)
-      : isLoadingCreate
-        ? () => {}
-        : onCreateClick,
-    primaryButtonDisabled: isPrimaryButtonDisabled || isLoadingCreate,
+    primaryButtonOnClick: isCreated ? () => TemplateService()?.openBladeAfterCreate(workflowName) : isCreating ? () => {} : onCreateClick,
```

Comment:
You should not check if it is in isCreating here because the button is already disabled so just have onCreateClick